x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -7,82 +7,58 @@
 import { i18n } from '@kbn/i18n';
 
 export enum AlertType {
-  ErrorRate = 'apm.error_rate',
+  ErrorCount = 'apm.error_rate', // ErrorRate was renamed to ErrorCount but the key is kept as `error_rate` for backwards-compat.
```

Comment:
I assumed changing the alert type from `apm.error_rate` to `apm.error_count` would cause issues for existing alerts. If this is not correct I'm more than happy to go through with the renaming.

Reply:
It's used as the `id`, so it's likely to be persisted one way or another. Possibly just duplicated rows in the Alerting UI but I think keeping the id the same here is the more reasonable choice.

Reply:
Thanks for pitching in. Will leave it as-is.